x-pack/plugins/stack_connectors/server/connector_types/openai/openai.test.ts

```diff
@@ -63,6 +63,10 @@ describe('OpenAIConnector', () => {
         apiUrl: 'https://api.openai.com/v1/chat/completions',
         apiProvider: OpenAiProviderType.OpenAi,
         defaultModel: DEFAULT_OPENAI_MODEL,
+        headers: {
```

Comment:
I'm not seeing a test where we don't pass `headers` in at all - seems like we should have a simple one anyway

Reply:
Cheers, added a test!